src/assets/scripts/aircraft/AircraftInstanceModel.js

```diff
@@ -904,12 +903,15 @@ export default class Aircraft {
 
         let isExpeditingString = '';
         if (expedite) {
-            isExpeditingString = 'and expedite';
+            isExpeditingString = ' and expedite';
```

Comment:
Yeah, the trailing space would cause more `"AAL123, descend and maintain 5000 , turn right heading 270" spacing issues.

That sounds good though, I'll get that done sometime later today.